odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/mapper/fhir/PatientMapper.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.odoo.openmrs.mapper.fhir;
+
+import com.ozonehis.eip.odoo.openmrs.mapper.ToFhirMapping;
+import com.ozonehis.eip.odoo.openmrs.model.Partner;
+import org.hl7.fhir.r4.model.Patient;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PatientMapper implements ToFhirMapping<Patient, Partner> {
+
+    @Override
+    public Patient toFhir(Partner odooDocument) {
+        Patient patient = new Patient();
+        patient.setId(odooDocument.getPartnerRef()); // This is the patient's ID in the FHIR server
```

Comment:
Don't you have to set the other fields?

Reply:
Right now this is not being used. I have left it just to be extended later on when required.